java-ast/tests/src/test/java/org/jboss/windup/ast/java/test/JavaEnumerationScanningTest.java

```diff
@@ -33,22 +35,25 @@ public void testEnumerationUsage()
         Assert.assertEquals(3, counter);
 
         ClassReference referenceTest = new ClassReference("java.nio.file.AccessMode.WRITE", "java.nio.file", "AccessMode", null,
-                    ResolutionStatus.RESOLVED, TypeReferenceLocation.ENUM_CONSTANT, 12, 4, 39, "AccessMode testMode=AccessMode.WRITE;\n");
-        Assert.assertTrue(references.contains(referenceTest));
-
-        referenceTest = new ClassReference("java.nio.file.AccessMode.READ", "java.nio.file", "AccessMode", null,
-                    ResolutionStatus.RECOVERED, TypeReferenceLocation.ENUM_CONSTANT, 31, 41, 21, "new EnumerationClassUsage(AccessMode.READ)");
+                    ResolutionStatus.RESOLVED, TypeReferenceLocation.ENUM_CONSTANT, 12, 4, 39, "AccessMode testMode=AccessMode.WRITE;");
         Assert.assertTrue(references.contains(referenceTest));
 
         referenceTest = new ClassReference("java.nio.file.AccessMode.WRITE", "java.nio.file", "AccessMode", null,
                     ResolutionStatus.RESOLVED, TypeReferenceLocation.ENUM_CONSTANT, 17, 30, 18, "EnumerationClassUsage.testAccessibleEnum(AccessMode.WRITE)");
         Assert.assertTrue(references.contains(referenceTest));
+
+        referenceTest = new ClassReference("java.nio.file.AccessMode.READ", "java.nio.file", "AccessMode", null,
+                ResolutionStatus.RESOLVED, TypeReferenceLocation.ENUM_CONSTANT, 31, 41, 21, "new EnumerationClassUsage(AccessMode.READ)");
+        Assert.assertTrue(references.contains(referenceTest));
     }
     
     @Test
     public void testEnumConstWithoutClassOnClasspath()
     {
-        List<ClassReference> references = ASTProcessor.analyze(getLibraryPaths(), getSourcePaths(),
+        Set<String> libraryPaths = new HashSet<>();
+        libraryPaths.add("src/test/resources/testclasses/enumeration/dependency/hibernate-search-engine-5.5.3.Final.jar");
```

Comment:
so do we not to try parse/analyze enums even they are not on classpath? This test is then wrongly named or am I wrong?